plugin/storage/integration/cassandra_test.go

```diff
@@ -80,6 +81,7 @@ func (s *CassandraStorageIntegration) initializeCassandra(t *testing.T) {
 	s.initializeDependencyReaderAndWriter(t, f)
 	t.Cleanup(func() {
 		require.NoError(t, f.Close())
+		testutils.VerifyGoLeaksOnce(t)
```

Comment:
wrong place - initializeCassandra is a utility function, not a test. VerifyGoLeaksOnce should be called from a real top-level test.